### PR TITLE
Add new Impulse quantity with relevant units

### PR DESCRIPTION
**Pull request by Sigvess92**
"ImpulseUnit" was generated with enum values out of order due to the constants being ordered alphabetically. Should I manually edit the values or is this OK? I see that the two most recent quantities were merged with the same "messy" structure...

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/ImpulseTests.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        protected override double DecanewtonSecondsInOneNewtonSecond => 0.1;
+        protected override double DecinewtonSecondsInOneNewtonSecond => 10;
+        protected override double KilogramMeterPerSecondsInOneNewtonSecond => 1;
+        protected override double KilonewtonSecondsInOneNewtonSecond => 1E-3;
+        protected override double MeganewtonSecondsInOneNewtonSecond => 1E-6;
+        protected override double MicronewtonSecondsInOneNewtonSecond => 1E6;
+        protected override double MillinewtonSecondsInOneNewtonSecond => 1E3;
+        protected override double NanonewtonSecondsInOneNewtonSecond => 1E9;
+        protected override double NewtonSecondsInOneNewtonSecond => 1;
+        protected override double PoundFootPerSecondsInOneNewtonSecond => 7.230657989877;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/ImpulseTests.cs:37]: Verified
7.230657989877
https://www.translatorscafe.com/unit-converter/en-US/impulse/

File: UnitsNet.Tests/CustomCode/ImpulseTests.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        protected override double SlugFootPerSecondsInOneNewtonSecond => 0.224735720691;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/ImpulseTests.cs:38]: Verified
0.224735720691
https://www.translatorscafe.com/unit-converter/en-US/impulse/

**Review by angularsen — CHANGES_REQUESTED** [review]
Thanks! I found a couple of things for you to look over, but other than that this looks great 👏

**Comment by Sigvess92** [comment]
@angularsen Thanks for the review. Updated PR.

File: Common/UnitDefinitions/Impulse.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    {
+      "SingularName": "PoundFootPerSecond",
+      "PluralName": "PoundFootPerSeconds",
+      "FromUnitToBaseFunc": "{x} / 7.230657989877",
+      "FromBaseToUnitFunc": "{x} * 7.230657989877",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "pound·s" ]
```
[the remark this answers is not in the document]
Comment (Sigvess92) [reply Common/UnitDefinitions/Impulse.json]: Good catch. I mixed up the two units "PoundFootPerSecond" and "PoundForceSecond" here. I added the missing unit and corrected the abbreviations.

**Comment by angularsen** [comment]
Thanks, the nuget should be out shortly.

[Release UnitsNet/5.0.0-rc010 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F5.0.0-rc010)

I forgot to explain, the messy enum values are intentional to reduce merge conflicts when multiple pull requests add units to the same quantity. It picks randomly from a range of available values. It then saves all allocated values to a separate file, to make sure they don't change when new units are added later, or when occasionally removing/renaming units as part of major version bumps.

**Pull request merged**